static/app/utils/replays/getCurrentUrl.tsx

```diff
@@ -20,11 +20,11 @@ function getCurrentUrl(
   const origin = initialUrl ? new URL(initialUrl).origin : '';
 
   const mostRecentNavigation = last(
-    navigationCrumbs.filter(({timestamp}) => +new Date(timestamp || 0) < currentTimeMs)
+    navigationCrumbs.filter(({timestamp}) => +new Date(timestamp || 0) <= currentTimeMs)
   )?.data?.to;
 
   if (!mostRecentNavigation) {
-    return initialUrl;
+    return origin;
```

Comment:
looks like this could still return an empty string, because on line 20 we've got the ternary ` a ? b : '';`